src/test/java/seedu/address/logic/commands/AddPersonCommandIntegrationTest.java

```diff
@@ -23,14 +23,14 @@ public class AddPersonCommandIntegrationTest {
 
     @BeforeEach
     public void setUp() {
-        model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```

Comment:
Tests are currently failing because the equal method of ModelManager also considers if there is a AppointmentBook now. We need to create `getTypicalAppointmentBook()` and refactor other test cases as well to support these incoming changes

Reply:
This comment is no longer relevant. I have just made changes in a recent commit to create sample AppointmentBooks to pass the test cases.